authors/apps/authentication/models.py

```diff
@@ -122,5 +122,3 @@ class BlacklistedToken(models.Model):
 
     class Meta:
         db_table = "blacklist"
-class BlacklistedToken(models.Model):
```

Comment:
Why did you remove this? Do we not need it anymore?

Reply:
The base branch(develop) had the class without the Meta class and the token property which during conflict someone forgot to keep the new version of the class and kept both. So I had to remove it